app/Http/Controllers/ProductCategoryController.php

```diff
@@ -13,32 +13,30 @@ public function show(ProductCategory $category, Request $request)
 		$sortBy_column = null;
         $orderBy = null;
 
-        switch ($request->query('sortBy')) {
-            case 'oldestFirst':
-	            $sortBy_column = 'created_at';
-	            $orderBy = 'asc';
-            break;
-
-            case 'cheapestFirst':
-	            $sortBy_column = 'price';
-	            $orderBy = 'asc';
-            break;
-
-            case 'expensiveFirst':
-	            $sortBy_column = 'price';
-	            $orderBy = 'desc';
-            break;
-
-            case 'popularFirst':
-			
-            case 'newestFirst':
-
-            default:
-	            $sortBy_column = 'created_at';
-	            $orderBy = 'desc';
-            break;
-        }
-
+    switch ($request->query('sortBy')) {
+        case 'oldestFirst':
+        $sortBy_column = 'created_at';
+        $orderBy = 'asc';
+        break;
+        case 'cheapestFirst':
+        $sortBy_column = 'price';
+        $orderBy = 'asc';
+        break;
+        case 'expensiveFirst':
+        $sortBy_column = 'price';
+        $orderBy = 'desc';
+        break;
+        case 'popularFirst':
+        $sortBy_column = 'views';
+        $orderBy = 'desc';
+        break;
+        case 'newestFirst':
+        default:
+        $sortBy_column = 'created_at';
+        $orderBy = 'desc';
+        break;
+    }
+    
```

Comment:
I think it's better to have the indents(indents on the breaks are nice too). It's easier to look at and more clearly defines the cases.

Reply:
Will do that future commits, I usually hit the reindent command on my IDE and that's what it did, so I left it as is.

Reply:
Sounds good 👍 Everything else looks good otherwise.